float32 boomP
float32 armP

float32 boomV
float32 armV

float32 boomI
float32 armI

int8 boomMode
int8 armMode
